.github/workflows/pre-release.yml

```diff
@@ -18,16 +18,12 @@ jobs:
     outputs:
       releases_created: ${{steps.release-please.outputs.releases_created}}
     steps:
-      - uses: google-github-actions/release-please-action@v3
+      - uses: googleapis/release-please-action@v4
         id: release-please
         with:
-          package-name: "@linc-technologies/ember-paper-password"
-          release-type: "node"
-          bootstrap-sha: "80dd68e536489b80855b0b638d675874f0f0b009"
-          versioning-strategy: "prerelease"
-          prerelease: true
-          skip-github-release: false
-          default-branch: "beta"
+          manifest-file: "./release-please-config.prerelease.json"
```

Comment:
really wanna move all configy files into `.github`

Reply:
Why not just have it as the default manifest file name and update the pre-release tag later?

Reply:
Everything has to be configured via a manifest file now :/

Reply:
Sure, but isnt the default manifest name `.release-please-manifest.json`, could leave the option out if we keep with the expected filename:
```
      # optional. customize path to .release-please-manifest.json
      manifest-file: .release-please-manifest.json
```

It'll be easier to diff the file if the name doesnt change and we just edit the contents (When setting prerelease false), else it'll show as a removal and addition of a file